### DISCUSSION TITLE
Emporia EVSE support

### DISCUSSION THREAD

**Discussion by AlaricEgli** [emporia-vue-local/esphome#266] [Ideas]
At times my Emporia EVSE doesn't respect scheduling and will stay on during on peak TOU hours. 

to get around this I'd like to take more control of the charger, But I'm not sure how related this project/product is to that one.

**Comment by flaviut** [comment]
Completely unrelated, nothing in this project can help you with that unfortunately.

**Reply by AlaricEgli** [reply]
Unfortunate. Thank you for the prompt feedback.

**Comment by dalklein** [comment]
> At times my Emporia EVSE doesn't respect scheduling and will stay on during on peak TOU hours.
> 
> to get around this I'd like to take more control of the charger, But I'm not sure how related this project/product is to that one.

Look into PyEmVue.  Unfortunately still using the cloud, but, my local raspberrypi sends commands to start/stop charging and change the power level.  (not changing the breaker rating power limit).  I'm using it to control my Emporia EVSE, to do excess solar consumption charging.
